modules/fargate/lambda/main.tf

```diff
@@ -0,0 +1,71 @@
+resource "random_string" "lambda_postfix_generator" {
```

Comment:
I think you should be able to use the `telia-oss/terraform-aws-lambda` module directly here and avoid the code duplication.